history/titles/c_lakeshire.txt

```diff
@@ -1,9 +1,13 @@
 # k_stormwind
 1.1.1={
-	liege=k_stormwind
 	law = cognatic_succession
 	law = succ_primogeniture
 }
+499.5.11={
+	liege=c_redridge_canyons
```

Comment:
Count title can't be vassal of another count.

Reply:
fixed